boards/posix/native_posix/doc/board.rst

```diff
@@ -519,6 +519,20 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**Display driver**:
+  A display driver is provided that will create a window on the host machine to
+  render the display content. Underlying the display driver is using the
+  `SDL2`_ library for rendering and as such requires a 32-bit version of the
+  SDL2 library installed on the host machine. Further there is a dependency on
+  pkg-config to correctly pickup the SDL2 install path and compiler flags.
+
+  On a Ubuntu 18.04 host system this results in the needs of installing the
```

Comment:
`On a Ubuntu 18.04 host system this results in the needs of installing the` seems a too convoluted. Maybe just "In Ubuntu 18.04 install the xx and yy packages,"? instead. 
But better if @dbkinder proposes.